### PR TITLE
refactor(editor): Replace mixed style of defineProps with the new style (no-changelog)

### PR DESCRIPTION
**Pull request by netroy**
## Summary
When defining props in components following the composition API we should use the recommended style, and avoid using `as PropType<*>`

## Review / Merge checklist

- [x] PR title and summary are descriptive

### CODE REVIEW

File: packages/editor-ui/src/components/executions/global/GlobalExecutionsList.vue [hunk 1]
```diff
@@ -13,24 +12,18 @@ import type { ExecutionSummary } from 'n8n-workflow';
...
+		total: number;
+		estimated: boolean;
+	}>(),
+	{
+		total: 0,
+		estimated: false,
 	},
-	filters: {
-		type: Object as PropType<ExecutionFilterType>,
-		default: () => ({}),
```
Comment (alexgrozav) [inline packages/editor-ui/src/components/executions/global/GlobalExecutionsList.vue:23]: Missed the default here.

File: packages/editor-ui/src/components/executions/global/GlobalExecutionsList.vue [hunk 2]
```diff
@@ -13,24 +12,18 @@ import type { ExecutionSummary } from 'n8n-workflow';
 import { useWorkflowsStore } from '@/stores/workflows.store';
 import { useExecutionsStore } from '@/stores/executions.store';
 
-const props = defineProps({
-	executions: {
-		type: Array as PropType<ExecutionSummary[]>,
-		default: () => [],
```
Comment (alexgrozav) [inline packages/editor-ui/src/components/executions/global/GlobalExecutionsList.vue:19]: Missed the default here.

**Review by alexgrozav — APPROVED** [review]
LGTM ✅

File: packages/editor-ui/src/components/executions/global/GlobalExecutionsList.vue (hunk 1 above)
Comment (netroy) [reply packages/editor-ui/src/components/executions/global/GlobalExecutionsList.vue:23]: these properties are required now, so no default needed. I checked the usage of many (but not all) of these components and updated some props to be required if we were passing them in from everywhere else in the code.

Comment (alexgrozav) [reply packages/editor-ui/src/components/executions/global/GlobalExecutionsList.vue:23]: Good, thanks for checking. Just wanted to make sure 👍

**Comment by janober** [comment]
Got released with [`n8n@1.47.0`](https://github.com/n8n-io/n8n/releases/tag/n8n%401.47.0)

**Comment by janober** [comment]
Got released with [`n8n@1.47.0`](https://github.com/n8n-io/n8n/releases/tag/n8n%401.47.0)

**Pull request merged**